src/modules/0.0.16/aws_secret/index.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  CreateSecretCommandInput,
+  DescribeSecretCommandInput,
+  GetSecretValueCommandInput,
+  SecretsManager,
+  PutSecretValueCommandInput,
+  UpdateSecretCommandInput,
+  paginateListSecrets,
+  SecretListEntry,
+  SecretsManagerClient,
+} from "@aws-sdk/client-secrets-manager";
+import {
+  AWS,
+  crudBuilder2,
+  paginateBuilder,
+} from "../../../services/aws_macros";
+import { Context, Crud2, Mapper2, Module2 } from "../../interfaces";
+import * as metadata from "./module.json";
+import { Secret } from "./entity/secret";
+
+async function createSecret(
+  client: SecretsManager,
+  input: CreateSecretCommandInput
+) {
+  const res = await client.createSecret(input);
+  if (res) {
+    const s: SecretListEntry = {
+      Name: res.Name,
+      Description: input.Description,
+    };
+    return s;
+  }
+  return undefined;
+}
+
+async function putSecretValue(
+  client: SecretsManager,
+  input: PutSecretValueCommandInput
+) {
+  const res = await client.putSecretValue(input);
+  if (res) {
+    return res;
+  }
+  return undefined;
+}
```

Comment:
I'm not asking for a change here, but aws_macros crudBuilder2 or crurBuilderFormat would have worked here. Maybe they are not that useful?